packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button.stories.ts

```diff
@@ -290,6 +323,16 @@ export const menuButtonColumn: StoryObj<MenuButtonColumnTableArgs> = {
                 disable: true
             }
         },
+        updateFavoriteColor: {
+            table: {
+                disable: true
+            }
+        },
+        currentRecord: {
+            table: {
+                disable: true
+            }
+        },
```

Comment:
@mollykreis  ah dang, sorry looked closer at the storybook and we need to disable menuState in the table as well
![image](https://github.com/user-attachments/assets/5e82bdad-94a5-477e-803c-0766c8466525)